frontend/src/pages/auth/forgot_password/index.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
```

Comment:
no need to import React on all pages. Importation has been done from the base index file. that covers the entire app